.vscode/settings.json

```diff
@@ -8,5 +8,12 @@
     "javascriptreact",
     "typescript",
     "typescriptreact"
-  ]
+  ],
+  "files.insertFinalNewline": true,
+  "[python]": {
+    "editor.tabSize": 4,
+    "editor.rulers": [
+      120
+    ],
```

Comment:
I think a better place for this might be the linting rules? Since this will only enforce with VSCode, you could have the situation where people have different settings in different IDEs, better have it be repo-wide.

Reply:
It is enforced by the linter, but just makes it nicer when in VScode, in particular the ruler

Reply:
Or rather whatever Prettier is for python, haha. It's more formatting than linting!

Reply:
Yes, that's `ruff`, and it's enforced by CI as well